bcdaworker/queueing/manager/alr.go

```diff
@@ -21,52 +31,139 @@ type alrQueue struct {
 	alrWorker worker.AlrWorker
 }
 
+// Max number of retries either set by ENV or default value of 3
+var maxRetry = int32(utils.GetEnvInt("BCDA_WORKER_MAX_JOB_NOT_FOUND_RETRIES", 3))
```

Comment:
You think it's good time to introduce a conf file/package for the worker? Similar to the [service config](https://github.com/CMSgov/bcda-app/blob/master/bcda/service/config.go)?